app/src/views/CountryOngoingActivitiesEmergencies/index.tsx

```diff
@@ -443,6 +446,15 @@ export function Component(props: BaseProps) {
         ? countryGroupedAppeal[clickedPointProperties.properties.iso3]
         : undefined;
 
+    if (countryResponse?.sovereign_state_id && countryResponse?.independent === false) {
+        return (
+            <Navigate
+                to={`/countries/${countryResponse.sovereign_state_id}/ongoing-activities/emergencies`}
```

Comment:
Please use `generatePath` util to generate the appropriate path. Here's an example https://github.com/IFRCGo/go-web-app/blob/2de417a37a36d1f49f4a80db38b159e3199c727c/app/src/views/Country/index.tsx#L101